script/component-migration/samples/AdditionalInfo.jsx

```diff
@@ -71,9 +73,7 @@ export const Sample3 = ({ className = 'sample' }) => (
 
 export const Sample4 = () => (
   <div className="vads-u-margin-y--3 available-connected-apps">
-    <AdditionalInfo
-      triggerText={`What other third-party apps can I connect to my profile?`}
-    >
+    <AdditionalInfo triggerText="What other third-party apps can I connect to my profile?">
```

Comment:
Would it be possible to change this to `va-additional-info` while you're making an update to this file? 🙏🏻  I think it should be a pretty straight forward swap out. You don't even need to import anything for it. 

https://design.va.gov/storybook/?path=/docs/components-va-additional-info--default

```js
<va-additional-info trigger="What other third-party apps can I connect to my profile?">
[slot content]
</va-additional-info>
```

Reply:
See previous comment, but I think this is supposed to be a sample of a legacy implementation with the React component, that you could then run the migration script on to migrate to the web component version? 🤷🏼